### ISSUE TITLE
Just a little typo for answer of "SetDHWHeatDelta"

### ISSUE THREAD

**Issue by Bogeyof** [heishamon/HeishaMon#256]
If I set "SetDHWHeatDelta" the answer is "set floor heat delta to.."

Code
```
unsigned int set_dhw_heat_delta(char *msg, unsigned char *cmd, char *log_msg) {
  unsigned int len = 0;
  String set_temperature_string(msg);

  byte request_temp = set_temperature_string.toInt() + 128;

  {
    char tmp[256] = { 0 };
    snprintf_P(tmp, 255, PSTR("set floor heat delta to %d"), request_temp - 128 );
    memcpy(log_msg, tmp, sizeof(tmp));
  }

  {
    memcpy_P(cmd, panasonicSendQuery, sizeof(panasonicSendQuery));
    cmd[99] = request_temp;
  }

  return sizeof(panasonicSendQuery);
}
```

Also the description in MQTT-Topics.md is wrong. This value must be set to "-12" - "-2". (For J-Model?).

MQTT-Topics.md
`SET20 | SetDHWHeatDelta | Set DHW heating delta in Kelvin | 1-15`

**Comment by MiG-41** [comment]
Yes , in H it is the same range : from -12 to -2 deg....

**Comment by Bogeyof** [comment]
Answer-Text is corrected in v3.0, may be also in v2.2. Both are a problem for me, cause JSON-Output is different and I think it´s a bit messed up...

**Comment by IgorYbema** [comment]
As noticed also in https://github.com/Egyras/HeishaMon/issues/257 I am fixing the json output now
